compiler/backend/reg_alloc/reg_allocScript.sml

```diff
@@ -1250,12 +1287,21 @@ val full_consistency_ok_def = Define`
     od
   od`
 
+val update_move_def = Define`
```

Comment:
I'm surprised this didn't show up in a compute lib.

Reply:
I've added this and a few others to the compute lib.